source/jormungandr/tests/proxy_realtime_integration_tests.py

```diff
@@ -190,16 +191,28 @@ def test_stop_schedule_with_rt_and_with_destination(self):
         assert stop_times[0]['date_time'][8:] == 'T080052'
         links = stop_times[0]["links"]
         assert len(links) == 1
-        assert links[0]['id'] == 'note:a9ce65528143e700d05dbfdffa78325c'
+        assert links[0]['id'] == 'note:539c0972470ff80a630a68e29d404c75'
 
         assert stop_times[1]['data_freshness'] == 'realtime'
         assert stop_times[1]['date_time'][8:] == 'T081352'
         links = stop_times[1]["links"]
         assert len(links) == 1
-        assert links[0]['id'] == 'note:a9ce65528143e700d05dbfdffa78325c'
+        assert links[0]['id'] == 'note:539c0972470ff80a630a68e29d404c75'
 
         notes = response['notes']
         assert len(notes) == 1
         assert notes[0]["type"] == "notes"
-        assert notes[0]["value"] == "C:S1"
-        assert notes[0]["id"] == "note:a9ce65528143e700d05dbfdffa78325c"
+        assert notes[0]["value"] == "Terminus"
+        assert notes[0]["id"] == "note:539c0972470ff80a630a68e29d404c75"
+
+    def test_stop_schedule_with_from_datetime_tomorrow(self):
+        query = self.query_template.format(sp='S42', dt='20160102T1100', data_freshness='', c_dt='20160103T1100')
```

Comment:
here your request is for a datetime that is yesterday, compared to current_datetime. Not sure that this is a problem, though, as I don't know how it is supposed to work in past, in future, and the delays.